flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientTest.java

```diff
@@ -138,6 +140,27 @@ public void testHistoryFile() throws Exception {
         }
     }
 
+    @Test
+    public void testSqlClientVerbose() throws Exception {
```

Comment:
Please verify this feature in `flink-table/flink-sql-client/src/test/resources/sql/set.q`. We don't add tests in `CliClientTest` unless it can't be tested by SQL scripts.